Managers/AudioMan.h

```diff
@@ -1,707 +1,446 @@
+
+		/// <summary>
+		/// Fades out playback of a specific sound.
+		/// </summary>
+		/// <param name="pSound">Pointer to the Sound to fade out playing. Ownership is NOT transferred!</param>
+		/// <param name="fadeOutTime"> The amount of time, in ms, to fade out over.</param>
+		void FadeOutSound(Sound *pSound, int fadeOutTime = 1000);
+#pragma endregion
+
+#pragma region Network Audio Handling
+		/// <summary>
+		/// Returns true if manager is in multiplayer mode.
+		/// </summary>
+		/// <returns>True if in multiplayer mode.</returns>
+		bool IsInMultiplayerMode() { return m_IsInMultiplayerMode; }
+
+		/// <summary>
+		/// Sets the multiplayer mode flag.
+		/// </summary>
+		/// <param name="value">Whether this manager should operate in multiplayer mode.</param>
+		void SetMultiplayerMode(bool value) { m_IsInMultiplayerMode = value; }
+
+		/// <summary>
+		/// Fills the list with sound events happened for the specified network player.
+		/// </summary>
+		/// <param name="player">Player to get events for.</param>
+		/// <param name="list">List with events for this player.</param>
+		void GetSoundEvents(int player, std::list<NetworkSoundData> & list);
+
+		/// <summary>
+		/// Adds the sound event to internal list of sound events for the specified player.
+		/// </summary>
+		/// <param name="player">Player for which the event happened.</param>
+		/// <param name="state">Sound state.</param>
+		/// <param name="hash">Sound file hash to transmit to client.</param>
+		/// <param name="distance">Sound distance.</param>
+		/// <param name="channel">Channel where sound was played.</param>
+		/// <param name="loops">Loops counter.</param>
+		/// <param name="pitch">Pitch value.</param>
+		/// <param name="affectedByPitch">Whether the sound is affected by pitch.</param>
+		void RegisterSoundEvent(int player, unsigned char state, size_t hash, short int distance, short int channel, short int loops, float pitch, bool affectedByPitch);
+
+		/// <summary>
+		/// Fills the list with music events happened for the specified network player.
+		/// </summary>
+		/// <param name="player">Player to get events for.</param>
+		/// <param name="list">List with events for this player.</param>
+		void GetMusicEvents(int player, std::list<NetworkMusicData> & list);
+
+		/// <summary>
+		/// Adds the music event to internal list of music events for the specified player.
+		/// </summary>
+		/// <param name="player">Player for which the event happened.</param>
+		/// <param name="state">Music state.</param>
+		/// <param name="filepath">Music file path to transmit to client.</param>
+		/// <param name="loops">Loops counter.</param>
+		/// <param name="position">Music playback position.</param>
+		/// <param name="pitch">Pitch value.</param>
+		void RegisterMusicEvent(int player, unsigned char state, const char *filepath, int loops, double position, float pitch);
+#pragma endregion
+
+	protected:
+		static Entity::ClassInfo m_sClass; ///! ClassInfo for this class.
```

Comment:
Pretty sure I mentioned that it should be `//!` not `///!`
This applies to member comments below too